pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -107,10 +107,10 @@ void processArguments() throws Exception {}
      */
     @Getter
     abstract class NamespaceCommand extends BaseCommand {
-        @Parameter(names = "--tenant", description = "The function's tenant")
+        @Parameter(names = "--tenant", description = "The tenant of functions")
```

Comment:
Why are we changing to this wording?  This config is the tenant of a particular Pulsar Function the user is trying to submit.  Its not the tenant of Pulsar Functions framework.  Changing the wording to this seems confusing to me

Reply:
How about "The tenant of a function" or "The tenant of a Pulsar Function"?

Reply:
"The tenant of a Pulsar Function" +1

Reply:
Thank you very much for your advice and comment.

Reply:
cool, will fix it

Reply:
> "The tenant of a Pulsar Function" +1

@jerrypeng PTAL again, thanks